shared/macros-bash.jinja

```diff
@@ -1126,6 +1126,21 @@ rpm --quiet -q "{{{ pkgname }}}"
 {{%- endif -%}}
 {{%- endmacro -%}}
 
+{{#
+  # Set rule CCE value
+  #
+  # This macro gets the var cce_identifiers from the environment created by the build scripts.
+  # The cce_identifiers is a dictionary that contains either the 'cce':'CCENUM' record for the
+  # product this remediation is being built for, or it is empty, in which case the string 'CCE' is used.
+  #}}
+{{%- macro set_cce_value() -%}}
+{{% if 'cce' in cce_identifiers -%}}
+cce="{{{ cce_identifiers['cce'] }}}"
+{{%- else -%}}
+cce="CCE"
```

Comment:
I have found some rules in the built  RHEL 8 data stream (ssg-rhel8-ds.xml) which have `cce="CCE"` in the remediation script even if these rules have CCEs. For example, the rule `accounts_password_pam_lcredit` has a RHEL8 CCE number assigned, but it doesn't propagate to this macro. Can it be related to the fact that this rule is templated?